component-starter-server/src/main/java/org/talend/sdk/component/starter/server/configuration/ServletConfigurer.java

```diff
@@ -60,6 +60,8 @@ public void doFilter(final ServletRequest servletRequest, final ServletResponse
             HttpServletResponse.class
                     .cast(servletResponse)
                     .addHeader("Content-Security-Policy", configuration.getCsp());
+            servletRequest.setCharacterEncoding("UTF-8");
```

Comment:
Is that normal to force the request encoding to UTF8 ? If the client is not UTF8 it will generate an encoding issue no ?